src/doom/d_main.c

```diff
@@ -246,8 +248,12 @@ boolean D_Display (void)
     {
 	R_RenderPlayerView (&players[displayplayer]);
 
+	// [crispy] Snow
+	if(crispy->snowflakes)
+	    V_DrawSnow();
+
         // [crispy] Crispy HUD
-        if (screenblocks >= CRISPY_HUD)
+        if (screenblocks >= CRISPY_HUD || crispy->snowflakes)
```

Comment:
So, does this redraw the whole status bar background or only the status bar numbers?

Reply:
Yes, it redrawing whole status bar